### PR TITLE
WIP: enable l3ha on 3 node ci envs.

### PR DESCRIPTION
**Pull request by nirajdp76**
Enables l3ha on example envs with 3 node deployments.

### CODE REVIEW

**Comment by nirajdp76** [comment]
ok to test

**Comment by nirajdp76** [comment]
ok to test

File: envs/example/ci-full-rhel/group_vars/all.yml [hunk 1]
```diff
@@ -17,6 +17,13 @@ etc_hosts:
 
 neutron:
   enable_external_interface: True
+  l3ha:
```
Comment (wangxian-ibm) [inline envs/example/ci-full-rhel/group_vars/all.yml]: I have a feeling that we will need to set `arp_responder: False` as well here for ping to work.

File: playbooks/tests/tasks/network.yml [hunk 2]
```diff
@@ -52,8 +52,8 @@
 
 # try ping internet for 5 minutes
   - name: neutron router can ping internet
-    shell: ROUTER_NS=$( ip netns show | grep qrouter- | awk '{print $1}' );
-           ip netns exec ${ROUTER_NS} ping -c 5 8.8.8.8
+    shell: DHCP_NS=$( ip netns show | grep qdhcp- | awk '{print $1}' );
+           ip netns exec ${DHCP_NS} ping -c 5 8.8.8.8
     when: neutron.l3ha.enabled is defined and neutron.l3ha.enabled|bool
```
Comment (wangxian-ibm) [inline playbooks/tests/tasks/network.yml]: The `when` statement here is unnecessary here unless we have a reason to not test ping when l3ha is not enabled.

**Comment by epwn** [comment]
ok to test

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
ok to test

**Comment by wangxian-ibm** [comment]
retest

**Comment by wangxian-ibm** [comment]
@nirajdp76 This is failing to reach the floating ip because it doesn't know where the master router is. I will discuss with you on the alternative fixes, as there are multiple.

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
closing this PR as is being replaces by https://github.com/blueboxgroup/ursula/pull/2605

**Pull request closed without merging**